router/router.go

```diff
@@ -151,6 +154,11 @@ func (r *chiWrapper) Mount(pattern string, h http.Handler) {
 	r.chi.Mount(pattern, h)
 }
 
+// Returns the value of a URL parameter
+func (r *chiWrapper) URLParam(req *http.Request, name string) string {
+	return chi.URLParam(req, name)
+}
```

Comment:
i don't get why this needs to be a method on the chiWrapper?
why not make this a freestanding function?

Reply:
It doesn't have to be, as it doesn't access the `chi` instance, but from a functional perspective it felt like the appropriate place. What location do you suggest? `router/helpers.go`?

Reply:
i feel like it could just be some code like this to reexport it?

```go
const URLParam = chi.URLParam
```

Reply:
I personally prefer having a method that we explicitly implement and which we can customise the behaviour of in the future, if needed.

Is this a blocker for you?

Reply:
you can replace a reexport with a freestanding function at any point, it will not break anyone except if the signature differs which is the same if it is a function now.